ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -1324,6 +1324,13 @@ private static Vector3 SrgbLinearToGamma(Vector3 vLinearColor)
 
         return vGammaColor;
     }
+    public static string AddSuffixToVmapName(string path)
+    {
+        var newVmapName = Path.GetFileNameWithoutExtension(path) + "_d" + Path.GetExtension(path);
+        var newVmapPath = NormalizePath(Path.Combine(Path.GetDirectoryName(path), newVmapName));
```

Comment:
I think this line needs to be removed. In fact, I wouldn't put this method in map extract at all, but rather modify the variable going into `FileName = Path.GetFileNameWithoutExtension(fileName)` in the save dialog.